### PR TITLE
Change some default settings

### PR DESCRIPTION
**Pull request by dreamer**
This PR is as much a change request as an opening to the discussion.

@krcroft Feel free to point which of these options conflict with your prefferred ones, or maybe some other ones you want changed?

### CODE REVIEW

**Comment by krcroft** [comment]
I like you choices.  They are are a universal fit for the capabilities of modern desktop OSes across Windows, Linux, and MacOS - without needing anything OS-specific. 

`opengl` make sense because it's much harder (if not impossible) to buy a modern machine _without_ hardware-accelerated OpenGL support than _with_. Likewise, the combination of the `sharp` shader with `aspect` correction  gives the closest approximation to the original DOS experience.

It also makes sense to retain the DOS-facing hardware-emulation settings based on wisdom from upstream: the needs of DOS games hasn't changed, and won't change - so lets stick with what's known to work.

File: src/dosbox.cpp [hunk 1]
```diff
@@ -446,15 +446,22 @@ void DOSBOX_Init(void) {
 	Pint->SetMinMax(0,10);
 	Pint->Set_help("How many frames DOSBox skips before drawing one.");
 
-	Pbool = secprop->Add_bool("aspect",Property::Changeable::Always,false);
-	Pbool->Set_help("Do aspect correction, if your output method doesn't support scaling this can slow things down!");
+	Pbool = secprop->Add_bool("aspect", Property::Changeable::Always, true);
+	Pbool->Set_help("Do aspect correction. Affects display modes using non-square\n"
+	                "pixels (such as 320x200 or 640x400), scaling them to maintain\n"
+	                "4:3 aspect ratio the most games were designed for.\n"
+	                "Change to false to use square pixels in all modes.");
```
Comment (krcroft) [inline src/dosbox.cpp]: "_Do aspect correction_" sounds a bit awkward, given there's a verb a the front and back of the sentence. We can switch to the active-voice by dropping _do_, and starting with either _correct_, or more specifically _scale_.  I've reordered the sentences:

"Scales the vertical resolution to produce a 4:3 display aspect ratio, matching that of original standard definition monitors for which DOS games were designed. This setting only affects video modes that use non-square pixels, such as 320x200 or 640x400; where as modes such as 640x480 and 800x600 will be displayed as-is. "

File: src/dosbox.cpp [hunk 2]
```diff
@@ -446,15 +446,22 @@ void DOSBOX_Init(void) {
...
-	                 "then the scaler will be used even if the result might not be desired.\n"
-					 "To fit a scaler in the resolution used at full screen may require a border or side bars,\n"
-					 "to fill the screen entirely, depending on your hardware, a different scaler/fullresolution might work.");
+	Pmulti->Set_help("Scaler used to enlarge/enhance low resolution modes.\n"
+	                 "If 'forced' is appended, then the scaler will be used even if\n"
+	                 "the result might not be desired.\n"
+	                 "To fit a scaler in the resolution used at full screen may\n"
+	                 "require a border or side bars, to fill the screen entirely,\n"
+	                 "depending on your hardware, a different scaler/fullresolution\n"
+	                 "might work.");
```
Comment (krcroft) [inline src/dosbox.cpp]: Reworded the last sentence:
Note that some scalers may use black borders to fit the image within your configured display resolution.  If this is undesirable, try either a different scaler or enabling fullresolution output.

File: src/misc/programs.cpp [hunk 3]
```diff
@@ -832,12 +835,12 @@ void PROGRAMS_Init(Section* /*sec*/) {
 
 	MSG_Add("PROGRAM_CONFIG_SECURE_ON","Switched to secure mode.\n");
 	MSG_Add("PROGRAM_CONFIG_SECURE_DISALLOW","This operation is not permitted in secure mode.\n");
-	MSG_Add("PROGRAM_CONFIG_SECTION_ERROR","Section %s doesn't exist.\n");
+	MSG_Add("PROGRAM_CONFIG_SECTION_ERROR", "Section \"%s\" doesn't exist.\n");
 	MSG_Add("PROGRAM_CONFIG_VALUE_ERROR","\"%s\" is not a valid value for property %s.\n");
 	MSG_Add("PROGRAM_CONFIG_PROPERTY_ERROR","No such section or property.\n");
-	MSG_Add("PROGRAM_CONFIG_NO_PROPERTY","There is no property %s in section %s.\n");
+	MSG_Add("PROGRAM_CONFIG_NO_PROPERTY", "There is no property \"%s\" in section \"%s\".\n");
```
Comment (krcroft) [inline src/misc/programs.cpp]: These extra quotes are a very helpful touch to differentiate the exact value being used. 
:+1:

File: src/dosbox.cpp (hunk 1 above)
Comment (dreamer) [reply src/dosbox.cpp]: Used exactly this description - feels way more natural than the sentence I copied/modified from dosbox wiki.

File: src/dosbox.cpp (hunk 2 above)
Comment (dreamer) [reply src/dosbox.cpp]: Replaced the last sentence, thanks! :)

**Comment by dreamer** [comment]
Thanks!

**Pull request merged**